specification/containerregistry/resource-manager/readme.md

```diff
@@ -4,31 +4,40 @@
 
 This is the AutoRest configuration file for ContainerRegistry.
 
-
-
 ---
+
 ## Getting Started
+
 To build the SDK for ContainerRegistry, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the ContainerRegistry API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-06-preview
+tag: package-preview-2019-12
```

Comment:
How about follow the `yyyy-MM-preview` pattern for the tag?

Reply:
I do not remember changing this file, this file was changed by  "OpenApi Hub" on behalf of me.

Reply:
Thanks for the info. We will log a bug for `OpenApi Hub`.